dbschema/seeds/007.language-engagements.edgeql

```diff
@@ -0,0 +1,66 @@
+with
+  engagementsJson := to_json('[
+   {
+      "status": "InDevelopment",
+      "startDateOverride": "2020-04-01",
+      "endDateOverride": "2020-06-30",
+      "language": "English",
+      "lukePartnership": true,
+      "openToInvestorVisit": true,
+      "projectName": "Misty Mountains"
```

Comment:
Can you put project name first, then language name. I know you were sorting alphabetically, but these two are the defining label & have the exclusivity constraint.

Also "language", but then "projectName", but both are names. I'd just drop the "name" suffix - it's implied.